core/src/main/java/org/apache/brooklyn/core/effector/BasicParameterType.java

```diff
@@ -44,6 +45,7 @@ public BasicParameterType(Map<?, ?> arguments) {
         if (arguments.containsKey("type")) type = (Class<T>) arguments.get("type");
         if (arguments.containsKey("description")) description = (String) arguments.get("description");
         if (arguments.containsKey("defaultValue")) defaultValue = (T) arguments.get("defaultValue");
+        if (arguments.containsKey("sensitive")) defaultValue = (T) arguments.get("sensitive");
```

Comment:
Is it possible for the value in the map to be null? If yes then use `Boolean.TRUE.equals(args.get(..))`.